packages/cli/src/commands/init.ts

```diff
@@ -120,6 +123,12 @@ export async function promptForConfig(
       active: "yes",
       inactive: "no",
     },
+    {
```

Comment:
I think this question should be moved down after tailwindConfig and if config exists, the default answer could be pre-populated based on config.